modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -218,10 +221,9 @@ If there are problems with caching the parameters and caching the metadata, try
 
 == Revert the Upgrade
 
-If it is necessary to revert to the previous version of Salesforce Connector, change the `salesforce-connector` dependency version `10.0.0` in the project's `pom.xml` to the previous version.
-
-You must update the project's `pom.xml` in Anypoint Studio.
+If it is necessary to revert to the previous version of Salesforce Connector, change the `salesforce-connector` dependency version `10.0.0` in the project's `pom.xml` file to the previous version.
```

Comment:
Do we need to clarify here what we mean by "previous" version? Version previously used in you environment or the version released just before the current version?